Create ui-c8y-1022-40-1-avoid-violating-csp-by-setting-base-url-when-parsing-svg-in.md (#3782)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1022-40-1-avoid-violating-csp-by-setting-base-url-when-parsing-svg-in.md

---
date: ""

> title: Fix content security policy issue in SCADA widget

Reviewer: ```suggestion
title: Fixed content security policy issue in SCADA widget
```

> title: Avoid violating CSP by setting `base` URL when parsing SVG in legacy SCADA widget (#10025) [GRAFT][release/cd] (#10060)

Reviewer 2: ```suggestion
title: Fix content security policy issue in SCADA widget
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-64731
version: 1022.40.1
---

> This change fixes an issue with SCADA widget that could have triggered browser's warning about suspicious behavior of the Cockpit (or other app where SCADA widget is displayed in a dashboard).

Reviewer 2: ```suggestion
The SCADA widget had an issue that could trigger a browser warning about suspicious behavior in applications like Cockpit where the widget is used in dashboards. With this fix, users should no longer see browser warnings related to content security policy violations caused by the SCADA widget.
```
